task_20/Oleg.Mikhailov/src/main/java/students/City.java

```diff
@@ -0,0 +1,52 @@
+package students;
+
+public class City {
+
+  String country;
+  String city;
```

Comment:
This field looks better as a "name", so you do not repeat the name of the class itself